[pull] primary from flybywiresim:primary (#384)
Valastiri/fbw-userguide: docs/fbw-a32nx/support/reported-issues.md

## Common MSFS Issues impacting all Aircraft

??? failure "Performance Degradation In-Flight"
    ### Performance Degradation In-Flight

    !!! tip ""
        *Affected versions: Stable, Development*
    
    ^^Description^^

>     Reports in the MSFS forums detail issues impacting FPS performance in the sim. Notably this occurs during flights that are longer than 2 hours but is not contained to this metric. You may see your normal FPS drop to <10 FPS or even single digits as a result of this issue.

Reviewer: ```suggestion
    Reports in the MSFS forums detail issues impacting FPS performance in the sim. Notably this occurs during flights that are longer than 2 hours but is not contained to this metric. You may see your normal FPS drop to <10 FPS as a result of this issue.
```

<10 == single digits

    ^^Possible Solution or Workaround^^

    *Under Investigation*

>     We will release an experimental version soon that contains some fixes to help mitigate this. Please join our Discord and follow our `#server-announcements` channel.

Reviewer: ```suggestion
    We have released a new experimental version that contains some fixes to help mitigate this. Please join our Discord and follow our `#server-announcements` channel.
```
